### PR TITLE
Create UX week (first week of product-build)

### PR DESCRIPTION
**Pull request by bradreeder**
🎉 

I've translated the [dropbox notes](https://paper.dropbox.com/doc/UX-Week-FAC10-mi880Uu0tl090P0vrFEND) into a UX-week folder in the coursebook following a common structure to other curriculum weeks.

* Created README.md containing menu of files in folder and overview of week.
* Created file on research afternoon task.
* Created reference to learning outcomes of product-build folder.
* Created files for user-persona workshop, design thinking workshop, google ventures monday, google ventures thursday.
* Added file on notes on students preparation for pitches + facilitators preparation for week.
* Added file on materials facilitators need to get before start of week.

Related #14, #88, #36

### CODE REVIEW

**Comment by bradreeder** [comment]
@harrygfox @nogainbar  Do you think we should merge in what we have then create an issue on the next steps, or shall we continue working on it here? 

I think we need to do the following:

* Update design thinking workshop file as it's not self-evident how to do the workshop from the timeline alone.
* @harrygfox maybe we should add your slides for your Monday talk? 
* Google Ventures Tuesday, Wednesday and Friday need more detailed descriptions of tasks involved. @harrygfox Did you use any resources that are not referenced in the dropbox?
* Add a suggested format for running the presentations.
* Update the week based upon the [feedback we got from students](https://github.com/foundersandcoders/london-curriculum/blob/master/stop-go-continue/fac-10/cohort-week-10.md).
* Add a reference to this week in the product-build once the other PRs are merged. #227 #238

**Comment by nogainbar** [comment]
@bradreeder this is brilliant, thank you very much!!!
I think we can merge this and keep on iterating on it. 

happy to work on this when im back:
- Update design thinking workshop file as it's not self-evident how to do the workshop from the timeline alone

not entirely clear about what you mean by this?
- Add a suggested format for running the presentations.
do you mean the presentations of the students?

**Comment by jsms90** [comment]
@bradreeder 
1. In the body of this PR, please reference the issues that refer to "UX week" and/or "week 10-xx" and/or "project buid". We want to make sure that anyone following those issues sees a reference to the fact that this PR has been made against those discussions.

2. > Do you think we should merge in what we have then create an issue on the next steps, or shall we continue working on it here?  

From the comments above, it looks like both reviewers are happy with the current content. So it might be best to get this merged and make your additions in another PR? Otherwise you're going to end up asking for multiple reviews from the same people. We may end up with lengthy discussions in a PR (like PR #103 - 76 comments) & makes it quite unclear as to when the work is actually finished.

I agree with @harrygfox that iterations on the version that you have already delivered should be done via the regular process of raising issues and submitting PRs against them. But it also sounds like you guys have more to add to these files, before it's a complete account of what was delivered to FAC10?

I would suggest removing "WIP" from the title & assigning one of the `master-reference` maintainers to merge? If you guys need to add more content before the FAC10 version is complete, please can you do that before we start iterating?

File: coursebook/week-UX/preparation.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+## Preparation
+
+### Students
+
+Go away and think about what you want to build within the scope of a 2 week build. If you want to pitch ideas then choose something you really know about - be the product owners / experts.
```
Comment (jsms90) [inline coursebook/week-UX/preparation.md]: Repeated 'build'

Maybe 
> what you want to **create** within the scope of a 2 week build

or
> what you want to build within the scope of a 2 week **project**

File: coursebook/week-UX/research-afternoon.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+## Research Topics
+
+#### Prototyping Tools
+  - What is a wireframe?
+  - What is a prototype?
+  - Can you come up with ideas for different methods for prototyping?
+#### User Testing http://www.sensible.com/rsme.html
```
Comment (jsms90) [inline coursebook/week-UX/research-afternoon.md]: Maybe put these links on a new line? Looks funny in the title for some reason

**Comment by bradreeder** [comment]
@jsms90 Noticed a lot of other formatting errors. Hold off merging, will get back to it when this week is through.

**Comment by des-des** [comment]
@bradreeder shall I look at this now, or wait until the next round of changes?

**Comment by bradreeder** [comment]
@des-des After next round of changes, will get to it this week. 👍

**Comment by des-des** [comment]
@bradreeder cool re-assign me!

**Comment by bradreeder** [comment]
Well, this is going to be fun.

**Comment by bradreeder** [comment]
@jsms90 give me your best shot. ;)

**Comment by jsms90** [comment]
@bradreeder :laughing: Please say your sarky humour is going to start making its way into github :wink:

**Comment by des-des** [comment]
@jsms90 needed second review

**Comment by jsms90** [comment]
@des-des Good point! Sorry :sweat:

**Pull request merged**